eazye.go

```diff
@@ -112,7 +112,7 @@ func GenerateSince(info MailboxInfo, since time.Time, markAsRead, delete bool) (
 type Email struct {
 	Message *mail.Message
 
-	From         *mail.Address   `json:"from"`
+	From         string          `json:"from"`
```

Comment:
Instead of making this a breaking change, could we expose this by adding a new `FromRaw` string?